lib/linter-plus.coffee

```diff
@@ -34,12 +32,10 @@ class Linter
     @subscriptions.add atom.workspace.onDidChangeActivePaneItem =>
       @commands.lint()
 
-    @subscriptions.add atom.config.onDidChange 'linter.showErrorTabLine', =>
-      @views.updateTabs()
-    @subscriptions.add atom.config.onDidChange 'linter.showErrorTabFile', =>
-      @views.updateTabs()
-    @subscriptions.add atom.config.onDidChange 'linter.showErrorTabProject', =>
-      @views.updateTabs()
+    for tabName of @views.tabs.tabPriorities
```

Comment:
Note to self: this makes little sense here, should probably move to `TabContainer`